Library/Formula/parse-cli.rb

```diff
@@ -0,0 +1,17 @@
+class ParseCli < Formula
+  homepage "https://parse.com"
+  url "https://www.parse.com/downloads/cloud_code/parse",
```

Comment:
When the version changes, we should just update the formula right? Or is there a catch?